ssh/ssh.go

```diff
@@ -237,6 +238,24 @@ func (ctx *SSHContext) ActivateConfiguration(host Host, configuration string, ac
 	return nil
 }
 
+func (sshCtx *SSHContext) GetBootID(host Host) (string, error) {
+	ctx, cancel := context.WithTimeout(context.TODO(), 5 * time.Second)
+	defer cancel()
+	cmd, err := sshCtx.CmdContext(ctx, host, "cat", "/proc/sys/kernel/random/boot_id")
+	if err != nil {
+		return "", err
+	}
+
+	bytes, err := cmd.CombinedOutput()
```

Comment:
Don't like `CombinedOutput()`. Outbursts from the ssh client like: `Warning: Permanently added <new-ip> to ssh known-hosts` end up in stderr - and thus in `bytes`, which has caused problems before when we depend on strict command output.

Reply:
Good point. Since it's difficult to combine redirecting stdout and stderr while also getting the output of the function, maybe we should transfer the file to localhost with scp instead?

Reply:
I guess you _could_ do that, but isn't this just a case where it's safer (and safe enough) to just read stdout and stderr into two different buffers, dump the former into `bytes` and dump the latter at morph stderr? I think we do something similar in other places.

Reply:
Sure. We can do that. However, I kinda feel it's really ugly to cat a file over SSH instead of just scp'ing it, although the latter requires temporary files, which is also _ick_.